### DISCUSSION TITLE
Did something change with the movies extras script recently?

### DISCUSSION THREAD

**Discussion by Jdiesel87** [RandomNinjaAtk/docker-radarr-extended#43] [General]
I have been using the docker for about a year now with the movies extras script set to single official trailer trailers only. It has been working great and placing the trailer in the /trailers sub directory renamed to the movie name with year. This was compatible with Emby.

On March 16, 2023 I can see that a bunch of new trailers were downloaded for all of my movies, sometimes multiple for each movie, and did not rename them. It's almost like the script is no longer respecting my docker values. I would really like to clean these up, it's going to take a while to delete all the new files for my library, but I don't want to spend the time if it will happen again.

**Comment by RandomNinjaAtk** [comment]
You can check the commit history here, a bug fix was recently implemented. Maybe a side effect...

Post your settings and some screenshots/log info.


Edit: I think I see what is causing it after a quick review. I'll push a bug fix for it soon. However it won't perform any sort of cleanup.

**Comment by RandomNinjaAtk** [comment]
This should fix the issue: https://github.com/RandomNinjaAtk/docker-radarr-extended/commit/7575195b6f07a1fdaf8e99b2445828f1eb71f39d

It is also going to attempt to perform a cleanup automatically, but its untested....

**Reply by Jdiesel87** [reply]
Thanks that seemed to do the trick. Appreciate the quick fix
